src/components/forms/checkbox/checkbox.stories.mdx

```diff
@@ -1,20 +1,23 @@
 import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs';
-import { Checkbox } from './checkbox'; 
+import { Checkbox } from './checkbox';
+import { getSampleController } from '../text-field-story-helper';
 
 <Meta
-  argTypes={{onChange: { action: 'onChange' }}}
+  argTypes={{ onChange: { action: 'onChange' } }}
   title="Components/Forms/Checkbox"
-  component={{ Checkbox }}/>
-
-> ⚠️ API not usable yet, design focused only
+  component={{ Checkbox }}
+/>
 
 # Checkbox
+
 Checkboxes are used for a list of options where the user may select multiple options, including all or none.
 
-<Canvas withSource='open'>
+Besides the `label` it receives props of the type `UseControllerProps`. The API of this component is based on the [controlled approach of react-hook-form](https://react-hook-form.com/api/usecontroller).
```

Comment:
Can you also add a variant where the error message is visible?

Reply:
I tried to add an error message, but I it seems that storybook has some troubles with executing forms. 

I was not able to get the form to work to add an error manually.

Reply:
I tried to add an error message, but I it seems that storybook has some troubles with executing forms. 

I was not able to get the form to work to add an error manually.

Reply:
Since this is not critical for the functionality of the form, I suggest to merge for now and to add the the error state story later, when I am able to do it